### PR TITLE
Feature Store guide

### PR DESCRIPTION
**Pull request by ionicsolutions**

### CODE REVIEW

File: docs/using-anovos/config_file.md [hunk 1]
```diff
@@ -1579,64 +1641,80 @@ autoencoder_latentFeatures:
...
+  - [📖 Writing CSV files](https://sparkbyexamples.com/pyspark/pyspark-read-csv-file-into-dataframe/)
+  - [📖 Writing Parquet files](https://sparkbyexamples.com/pyspark/pyspark-read-and-write-parquet-file/)
+  - [📖 Writing Avro files](https://sparkbyexamples.com/spark/read-write-avro-file-spark-dataframe/)
+
+## 📑 `write_feast_features`
+
+🔎 _Corresponds to [`??.??`](../api/??#??)_
+
+📖 _For details, see the_
+_[Feature Store Integration documentation](https://sparkbyexamples.com/pyspark/pyspark-read-csv-file-into-dataframe/)_
```
Comment (ionicsolutions) [inline docs/using-anovos/config_file.md:1720]: @mathiaspet We need a complete list of all possible configuration options

File: docs/using-anovos/config_file.md [hunk 2]
```diff
@@ -1579,64 +1641,80 @@ autoencoder_latentFeatures:
...
+  For more information on available configuration options,  see the following external
+  documentation:
+
+  - [📖 Writing CSV files](https://sparkbyexamples.com/pyspark/pyspark-read-csv-file-into-dataframe/)
+  - [📖 Writing Parquet files](https://sparkbyexamples.com/pyspark/pyspark-read-and-write-parquet-file/)
+  - [📖 Writing Avro files](https://sparkbyexamples.com/spark/read-write-avro-file-spark-dataframe/)
+
+## 📑 `write_feast_features`
+
+🔎 _Corresponds to [`??.??`](../api/??#??)_
```
Comment (ionicsolutions) [inline docs/using-anovos/config_file.md:1717]: TODO once the API docs have been re-generated

File: docs/using-anovos/feature_store.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+pip install feast
+```
+
+Next, we'll instantiate a new Feast repository:
+
+```bash
+feast init anovos_repo
+```
+
+🤓  _Note: You can also use an existing repository._
```
Comment (ionicsolutions) [inline docs/using-anovos/feature_store.md:36]: @mathiaspet What does a user have to keep in mind? Is there a page in the Feast docs we can point them to? (I believe they need to take care of versioning the feature definitions via Git?)

File: docs/using-anovos/feature_store.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    create_timestamp_col: 'create_time_col'        # ...
+  feature_view:
+    name: 'income_view'                            # ...
+    owner: 'view@owner.com'                        # the view owner registered in Feast
+    ttl_in_seconds: 3600                           # ...
+    create_timestamps: True                        # ...
+  feature_service:                                 # ...
+    name: 'income_feature_service'                 # ...
+  file_configs:
+    mode: overwrite                                # ...
```
Comment (ionicsolutions) [inline docs/using-anovos/feature_store.md:66]: @mathiaspet Could you please add a brief description for the relevant lines?

File: docs/using-anovos/feature_store.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+Let's break this down!
```
Comment (ionicsolutions) [inline docs/using-anovos/feature_store.md:69]: @mathiaspet Input, please :-)

File: docs/using-anovos/feature_store.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+To verify that the features have been loaded correctly, you can check them using Feast's UI.
+Run
+
+```bash
+feast ui
+```
+
+and access the Feast UI at [`http://127.0.0.1:8888`](http://127.0.0.1:8888).
+
+There, ...
```
Comment (ionicsolutions) [inline docs/using-anovos/feature_store.md:130]: @mathiaspet Where do I go? What do I check for?

File: docs/using-anovos/feature_store.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+entity_df = pd.DataFrame.from_dict(
+  {
+      "ifa": ['', '', '' ''], # add some ids
+      "event_time": [
+          datetime(2022, 5, 31, 14, 59, 42), # add datetiem entries corresponding to timestamps of the generated features
+          datetime(2022, 5, 31, 8, 12, 10),
+          datetime(2022, 5, 31, 16, 40, 26),
+          datetime(2022, 5, 31, 15, 1, 12),
+      ],
+  }
```
Comment (ionicsolutions) [inline docs/using-anovos/feature_store.md:152]: Is there an easier way to do this? Can't I just say "Give me everything between A and B?"

File: docs/using-anovos/feature_store.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+    features=feature_service, entity_df=income_entities
+).to_df()
+print(income_features_by_service_df.head())
+
+# Train your model
+
+...
+```
+
+## Integrating _Anovos_ with other feature stores
```
Comment (sinuochen) [inline docs/using-anovos/feature_store.md:206]: @mathiaspet Seems that this heading contains no content.

Comment (ionicsolutions) [reply docs/using-anovos/feature_store.md:206]: Yes, we're aware of that :-) We're waiting for the PR in the main repo to be closed to be finally able to generate the API docs and finish up this PR

**Review by mathiaspet — APPROVED** [review]

**Pull request merged**